lambda_policy.tf

```
@@ -30,6 +30,7 @@ data "aws_iam_policy_document" "lambda" {
       "logs:DeleteLogGroup",
       "logs:DescribeLogGroups",
       "logs:ListTagsLogGroup",
+      "logs:ListTagsForResource",
       "logs:PutRetentionPolicy",
```

Preserve alphabetization:
```suggestion
      "logs:ListTagsForResource",
      "logs:ListTagsLogGroup",
      "logs:PutRetentionPolicy",
```